telegram/__init__.py

```diff
@@ -92,7 +93,6 @@
 from .shippingquery import ShippingQuery
 from .webhookinfo import WebhookInfo
 from .gamehighscore import GameHighScore
-from .videonote import VideoNote
```

Comment:
well, what every git voodoo happened here, I prefer it not to be part of this PR